docs/getting-started/installation.md

```diff
@@ -55,27 +66,31 @@ To get the full gist of this, let's run a minified version of the - [Quickstart
 mkdir data && cd data
 
 # download wikipedia dataset files
-curl -o wikipedia_index_config.json https://raw.githubusercontent.com/quickwit-inc/quickwit/main/examples/index_configs/wikipedia_index_config.json
+curl -o wikipedia_index_config.yaml https://raw.githubusercontent.com/quickwit-inc/quickwit/main/config/tutorials/wikipedia/index-config.yaml
 curl -o wiki-articles-10000.json https://quickwit-datasets-public.s3.amazonaws.com/wiki-articles-10000.json
+```
+
+To expose the rest API outside of the docker container, we need to customize the default configuration provided by Quickwit. Download it, uncomment and update the searcher config section by setting the `rest_listen_address` to `0.0.0.0`. 
```

Comment:
Would be nice to be able to provide an config file with the right `rest_listen_address`. 
What I would do is to put a `quickwit_docker.yaml` with `rest_listen_address` to `0.0.0.0`.
Then we can just tell the user to download the file.

Reply:
I thought about it but, I was like it would be many files. I should have pinged you about it.
Thanks I will do that.